instance.php

```diff
@@ -51,3 +56,17 @@ function current_snapshot_uuid() {
 		return $customize_snapshot_uuid;
 	}
 }
+
+/**
+ * Returns whether it is back compat or not.
+ *
+ * @return bool is compat.
+ */
+function is_back_compat() {
+	// Fix in case version contains 'src' for example 4.7-src in that case php version compare fails to compare correctly.
+	$wp_version = get_bloginfo( 'version' );
+	if ( $pos = strpos( $wp_version, 'src' ) ) {
+		$wp_version = substr( $wp_version, 0, $pos - 1 );
+	}
+	return version_compare( $wp_version, '4.7', '<' );
```

Comment:
Why not use the global `$wp_version` instead of trying to check the string. The whole method could be rewritten like:

```PHP
function is_back_compat() {
	global $wp_version;

	return version_compare( $wp_version, '4.7', '<' );
}
```